cogs/minecraft_data.py

```diff
@@ -5,52 +5,38 @@
 from discord.ext import commands, tasks
 import voxelbotutils as utils
 
+import json
 
 class MinecraftData(utils.Cog):
 
     # https://mcapi.us/server/status?ip=149.202.84.162&port=25588
-    MINECRAFT_API = "https://mcapi.us/server/status"
+    MINECRAFT_API = "https://api.mcsrvstat.us/1/vfl.mcserver.at"
     SERVER_IP = "vfl.mcserver.at"
     SERVER_PORT = 25565
     MINECRAFT_MESSAGE = (716321756171862048, 721771100790325321)  # ChannelID, MessageID
 
     def __init__(self, bot:utils.Bot):
         super().__init__(bot)
-        self.server_status_update.start()
 
-    def cog_unload(self):
-        self.server_status_update.cancel()
-
-    @tasks.loop(minutes=10)
-    async def server_status_update(self):
-        """Updates the Minecraft status message"""
-
-        # Get message
-        channel = self.bot.get_channel(self.MINECRAFT_MESSAGE[0])
-        message = await channel.fetch_message(self.MINECRAFT_MESSAGE[1])
-
-        # Get data
-        async with self.bot.session.get(self.MINECRAFT_API, params={"ip": self.SERVER_IP, "port": self.SERVER_PORT}) as r:
+    @utils.command(aliases=['mcstats', 'mcserver', 'mcserverstats'])
+    @commands.bot_has_permissions(send_messages=True)
+    async def server_status_update(self, ctx:utils.Context):
+        """Shows the status of the VFL minecraft server"""
+        async with self.bot.session.get(self.MINECRAFT_API) as r:
```

Comment:
This will only get the status of the VFL Minecraft server. The point of separating the API and the IP is so that we can use them separately. More apt would be something like `f"{self.MINECRAFT_API}/{self.SERVER_IP}"`, which _looks_ like a bit of a pain but is better practise.